### PR TITLE
Rename `array` to `tuple`

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Rename `array` to `tuple`

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon I'm trying to add last changes from the `eo-runtime` to `objectionary/home` and get the next exception:

```
Failed tests:   EOorg.EOeolang.EOpositive_infinity_gt_intTest.testWorks(): EOorg.EOeolang.EOstringν200="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOtry_memory_updateTest.testWorks(): (..)
  EOorg.EOeolang.EOpositive_infinity_not_eq_negative_infinityTest.testWorks(): EOorg.EOeolang.EOstringν2472="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_plus_multiple_numberTest.testWorks(): EOorg.EOeolang.EOstringν2608="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_lt_negative_infinityTest.testWorks(): EOorg.EOeolang.EOstringν2664="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnan_as_bytes_is_validTest.testWorks(): EOorg.EOeolang.EOstringν3091="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_not_lte_nanTest.testWorks(): EOorg.EOeolang.EOstringν4249="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_not_lt_nanTest.testWorks(): EOorg.EOeolang.EOstringν4423="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_lte_floatTest.testWorks(): EOorg.EOeolang.EOstringν4874="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOpositive_infinity_lte_positive_infinityTest.testWorks(): EOorg.EOeolang.EOstringν8890="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_minus_positive_intTest.testWorks(): EOorg.EOeolang.EOstringν8938="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
  EOorg.EOeolang.EOnegative_infinity_lte_negative_infinityTest.testWorks(): EOorg.EOeolang.EOstringν9128="The 1th argument of 'float.div' is invalid: division by zero is infinity"SF
```
Do you know what could be a cause of the problem?

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

File: tests/org/eolang/seq-tests.eo [hunk 1]
```diff
@@ -25,8 +25,12 @@
 +home https://github.com/objectionary/eo
 +junit
 +package org.eolang
-+version 0.29.0
++version 0.29.1
 
+# @todo #1299:60min Implement test that will
```
Comment (Graur) [inline tests/org/eolang/seq-tests.eo:30]: @volodya-lombrozo You forgot to remove this todo

File: tests/org/eolang/runtime-tests.eo [hunk 2]
```diff
@@ -307,6 +307,9 @@
       foo.y.<
     $.equal-to TRUE
 
+# @todo #1502:30min For some reason this test doesn't work any more,
```
Comment (Graur) [inline tests/org/eolang/runtime-tests.eo:310]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/tuple.eo [hunk 3]
```diff
@@ -23,22 +23,26 @@
...
 +home https://github.com/objectionary/eo
 +package org.eolang
-+rt jvm org.eolang:eo-runtime:0.29.0
-+version 0.29.0
++rt jvm org.eolang:eo-runtime:0.29.1
++version 0.29.1
 
-[] > array
-  # Create empty array
+# @todo #1060:30min The method .empty is not nessesary.
```
Comment (Graur) [inline objects/org/eolang/tuple.eo:29]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/switch.eo [hunk 4]
```diff
@@ -41,6 +41,9 @@
 #     "this value will be skipped"
 #
 # This object returns value of only first truly statement.
+# @todo #1812:30min. Remake "switch" without using "QQ.collections.list".
```
Comment (Graur) [inline objects/org/eolang/switch.eo:44]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/rust.eo [hunk 5]
```diff
@@ -23,9 +23,13 @@
 +architect yegor256@gmail.com
 +home https://github.com/objectionary/eo
 +package org.eolang
-+rt jvm org.eolang:eo-runtime:0.29.0
-+version 0.29.0
++rt jvm org.eolang:eo-runtime:0.29.1
++version 0.29.1
 
+# @todo #1161:30min Complete implementation of this
```
Comment (Graur) [inline objects/org/eolang/rust.eo:29]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/ram.eo [hunk 6]
```diff
@@ -23,10 +23,14 @@
 +architect yegor256@gmail.com
 +home https://github.com/objectionary/eo
 +package org.eolang
-+rt jvm org.eolang:eo-runtime:0.29.0
-+version 0.29.0
++rt jvm org.eolang:eo-runtime:0.29.1
++version 0.29.1
 
 # Random-access memory (RAM).
+# @todo #684:30min We need to add ram.ram-slice.slice object
```
Comment (Graur) [inline objects/org/eolang/ram.eo:30]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/positive-infinity.eo [hunk 7]
```diff
@@ -61,57 +59,90 @@
...
-    ^.plus (x.neg) > @
+  # @todo #1806:30min. Make positive-infinity.minus and negative-infinity.minus
+  #  worked with multiple arguments without using objects from eo-collections.
+  #  See positive-infinity.plus and negative-infinity.plus as an example
+  [x...] > minus
+    ^.plus ((x.at 0).neg) > @
 
   # Quotient of the division of $ by x
-  [x] > div
+  # @todo #1806:30min. Make positive-infinity.div and negative-infinity.div
```
Comment (Graur) [inline objects/org/eolang/positive-infinity.eo:133]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/positive-infinity.eo [hunk 8]
```diff
@@ -61,57 +59,90 @@
...
     negative-infinity > @
 
-  # Sign of $ (-1.0, 0.0, or 1.0)
-  [] > signum
-    1.0 > @
-
   # Difference between $ and x
-  [x] > minus
-    ^.plus (x.neg) > @
+  # @todo #1806:30min. Make positive-infinity.minus and negative-infinity.minus
```
Comment (Graur) [inline objects/org/eolang/positive-infinity.eo:126]: @volodya-lombrozo You forgot to remove this todo

File: objects/org/eolang/positive-infinity.eo [hunk 9]
```diff
@@ -61,57 +59,90 @@
         nan.as-bytes
 
   # Multiplication of $ and x
-  [x] > times
+  # @todo #1806:30min. Make positive-infinity.times and negative-infinity.times
```
Comment (Graur) [inline objects/org/eolang/positive-infinity.eo:62]: @volodya-lombrozo You forgot to remove this todo

**Review by Graur — CHANGES_REQUESTED** [review]

File: tests/org/eolang/seq-tests.eo (hunk 1 above)
Comment (volodya-lombrozo) [reply tests/org/eolang/seq-tests.eo]: @Graur Thank you. Totally forgot about it. Done.

**Review by yegor256 — APPROVED** [review]
ok

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@volodya-lombrozo done

**Comment by yegor256** [comment]
@volodya-lombrozo done

**Pull request merged**